README.md

```diff
@@ -70,8 +70,8 @@ so as not overwrite the production `shis-data` files in Azure.
 The output JSON will be an array of objects in the format shown in the [Sample SHIS Data](sample-shis-data.json) file.
 Most of the fields are self-explanatory, the three that require further explanation are `id`, `location`, and `venueType`.
 
-`id` is only used internally. This is the Serco ID retrieved from Syndication with the filename prefixed to avoid collisions, i.e. `shis-data-19595086`.
-The Serco ID for a record may change when data is updated.
+`id` is only used internally. This is an ID retrieved from Syndication with the filename prefixed to avoid collisions, i.e. `shis-data-19595086`.
```

Comment:
This will need updating to correspond with the changes to the id not prepending the name of the file.